app/containers/IntentService.jsx

```diff
@@ -0,0 +1,102 @@
+/** @jsx h */
+import { h, Component } from 'preact'
+
+import Loading from '../components/Loading'
+
+export default class IntentService extends Component {
+  constructor (props, context) {
+    super(props, context)
+    this.store = context.store
+
+    const {window} = props
```

Comment:
Actually I am not sure about this one. What I want do achieve is to access the object `window` in my component, but I hate accessing it directly, so I prefered here pass it as a parameter.

The thing is, the window object is needed for creating an instance service with `cozy.client.intents.createService(intent, window)`, but having coded a real intent now I feel that all the stuff related to window should be directly done into `cozy.client.intents.createService()` like it's done here for example https://github.com/cozy/cozy-client-js/blob/master/src/auth_v2.js#L6